field_map.go

```diff
@@ -201,10 +225,31 @@ func (m *FieldMap) Clear() {
 
 // DeleteTag removes a tag's value from field map, if present
 func (m *FieldMap) DeleteTag(tag Tag) {
+	m.rwLock.Lock()
+	defer m.rwLock.Unlock()
 	delete(m.tagLookup, tag)
 }
 
+// CopyInto overwrites the given FieldMap with this one
+func (m *FieldMap) CopyInto(to *FieldMap) {
```

Comment:
Is `CopyInto` used anywhere?

Reply:
nvm, missed your comment in PR description that we are simply copying the commit from original repo.